code/modules/vore/fluffstuff/custom_clothes_vr.dm

```diff
@@ -806,6 +806,13 @@
 	rolled_sleeves = 0
 	starting_accessories = list(/obj/item/clothing/accessory/black)
 
+//Kisukegema:Kisuke `the nerd` Gema
+/obj/item/clothing/glasses/sunglasses/omnihud/kamina
+	name = "Kamina glasses"
+	icon_state = "kamina"
+	desc = "ROW ROW, FIGHT THE POWER."
+	see_invisible = SEE_INVISIBLE_LEVEL_TWO //DO THE IMPOSSIBLE, SEE THE INVISIBLE
```

Comment:
Oh hey, cool.

In that case, I might look into something that _does_ have the means of detecting them 'cause right now things like thermals dont. To avoid the problem we used to have of medihuds and sechuds instantly spotting them (a thing half the crew are wearing by default at all times) , I'd probably make them _specifically_ only an anti-cloaker counter and not combine them with thermals, HUDs or the like.

but that's a separate thing and not something for a fluff RD-hud :p